resteasy-jaxrs/src/main/java/org/jboss/resteasy/plugins/server/servlet/HttpServlet30Dispatcher.java

```diff
@@ -17,7 +17,11 @@
  * @author <a href="mailto:bill@burkecentral.com">Bill Burke</a>
  * @version $Revision: 1 $
  */
-@WebServlet(asyncSupported = true, value="")
+// Note. Setting value to "/RESTEASY_HttpServlet30Dispatcher" frees up the root "/" context
+//       to serve static content. The path "/RESTEASY_HttpServlet30Dispatcher" will be installed
+//       and it will lead to HttpServlet30Dispatcher, but no resources will be there and status
+//       404 will be returned.
+@WebServlet(asyncSupported = true, value="/RESTEASY_HttpServlet30Dispatcher")
```

Comment:
Does this change really belong to this fix? IOW, why is it needed?

Reply:
Looks like this is already in 3.9 because of https://github.com/resteasy/Resteasy/pull/2137/files , so it will go away when merging.